tap_salesforce/salesforce/rest.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, sf):
         self.sf = sf
 
     def query(self, catalog_entry, state):
-        start_date = self.sf.get_start_date(state, catalog_entry)
+        start_date = self.sf.get_start_date(state, catalog_entry, with_lookback=True)
```

Comment:
Remove `with_lookback` because it should not exist as a parameter in the new implementation

Reply:
Removed `with_lookback` param from all places.